code/game/objects/items/weapons/cigs.dm

```diff
@@ -100,58 +107,162 @@ LIGHTERS ARE IN LIGHTERS.DM
 	if(!lit)
 		light("<span class='warning'>[src] is lit by the flames!</span>")
 
-/obj/item/clothing/mask/cigarette/welder_act(mob/user, obj/item/I)
-	. = TRUE
-	if(I.tool_use_check(user, 0)) //Don't need to flash eyes because you are a badass
-		light("<span class='notice'>[user] casually lights [src] with [I], what a badass.</span>")
+/obj/item/clothing/mask/cigarette/proc/lighter_interaction(obj/item/I, mob/user, mob/living/carbon/target)
+	if(lit)
+		to_chat(user, "<span class='warning'>\The [initial(name)] is already lit!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>[initial(name)] is already lit!</span>")
```
Please test that this properly appends `The` in front

Reply:
`\The` is required, otherwise you get messages like "cigarette is already lit!" and "premium Havanan cigar is already lit!"

It's because I'm using `initial(name)` instead of `[src]` so that the message isn't "The lit cigarette is already lit!"